### DISCUSSION TITLE
Looks like /v3/public/oauth/token (maybe more?) stopped responding properly to http/2 requests around 2023-12-29 04:57 UTC

### DISCUSSION THREAD

**Discussion by bytheway** [etsy/open-api#1180] [Q&A]
My API client crashed and threw continuous errors starting around 2023-12-29 04:57 UTC tonight.  Using postman and debugging I tracked it down to http/2 requests to /v3/public/oauth/token now are returning http status 200 success, but with an empty body, so no token is returned.

I hardcoded my client to only make http/1.1 requests for now, which is still working.

**Comment by pawcik** [comment]
I'm not sure if it is related to just the `/v3/public/oauth/token` request. In my case, I randomly get an empty response (with 200 status code) to various v3 apis (with valid token).

The responses with status code 200 are getting this headers:

```
HTTP/1.1 200 OK
Connection: keep-alive
Content-Length: 0
Content-Type: text/html; charset=UTF-8
Date: Fri, 29 Dec 2023 12:15:40 GMT
Server: Apache
Set-Cookie: user_prefs=<MASKED>; expires=Sat, 28 Dec 2024 12:15:40 GMT; Max-Age=31536000; path=/; domain=.etsy.com 
```
